packages/vue/src/cypress/FInteractiveTable.pageobject.ts

```diff
@@ -147,7 +157,34 @@ export class FInteractiveTablePageObject implements BasePageObject {
                 throw Error("Invalid order");
         }
 
-        const column = this.headerRowItem().tableRowHeaderContent().eq(index);
+        const column = cy.get(`${this.selector} thead th:nth(${index})`);
         return column.find(`svg.${iconName}`);
```

Comment:
Kanske kombinera dessa selectorer direkt nu när du ändå petar på raderna? [Undvik find(..)](https://designsystem.forsakringskassan.se/latest/gettingstarted/contribute-to-fkds/testing.html#undvik_find)